qpc_tools/messages.py

```diff
@@ -26,6 +26,10 @@
 SERVER_INSTALL_OFFLINE_FILES_HELP = 'Specify the path to the server offline files'
 SERVER_INSTALL_VERSION_HELP = 'Specify the server version to install '\
     '(defaults to latest release)'
+SERVER_INSTALL_SERVER_IMAGE_HELP = 'Sets the name of the server image. Defaults to ' \
+    'the latest discover-server on redhat.registry.io'
+SERVER_INSTALL_DB_IMAGE_HELP = 'Sets the name of the DB image. Defaults to ' \
```

Comment:
Same comment as above (Change `Sets` to `Set` and our DB image name defaults to postgres - i don't think it pulls from redhat.registry.io upstream.)

Reply:
_ignore the second part of this since it should be from the downstream perspective_